Formula/monkeysphere.rb

```diff
@@ -15,6 +16,8 @@ class Monkeysphere < Formula
   end
 
   depends_on "gnu-sed" => :build
+  depends_on "bash" # Apple's BASH 3.2 is insufficient, BASH 4.X features are used
```

Comment:
This doesn't swap out their default shell unless they configure it to do so.

Reply:
> This doesn't swap out their default shell unless they configure it to do so.

Actually, this is true, it will not change the user shell (I've tried it on both Catalina and Mojave), sorry for the confusion.

Nevertheless, it will change a result for `/usr/bin/env bash`.

Reply:
I think `bash` 4 is backwards compatible, so the only case where this would be an issue is if you are developing a script which should run on system `bash` and you accidentally introduce newer features due to this.

Reply:
This is an argument for a `keg_only` `bash`.

Reply:
I thought `bash` was *not* `keg_only` for a reason. Or do you mean adding a versioned `keg_only` `bash@4`?

Reply:
If the desire is to allow packages like Monkeysphere which require bash 4.0+ without changing the bash which will be invoked on the from the path then some form of keg only bash will be needed. Making the existing bash formula keg_only would make a lot of people, like me, unhappy because we are already using it as our primary shell for better Linux compatibility.

Reply:
> I thought `bash` was _not_ `keg_only` for a reason. Or do you mean adding a versioned `keg_only` `bash@4`?

@reitermarkus These decisions are all somewhat arbitrary. I'd happily see all system duplicates be keg-only, personally.

We're not going to add multiple versions of Bash with different statuses.



> Making the existing bash formula keg_only would make a lot of people, like me, unhappy because we are already using it as our primary shell for better Linux compatibility.

You can add a single path to your PATH and it'd work the same as before.

Reply:
With the most recent changes the monkeysphere formula no longer cares whether bash is on the path or is keg only. This seems sufficient for this PR.

Can the decisions about whether to make bash formula keg_only and/or add a caveat to the bash formula about path alteration be handled on a separate issue/PR? I have wanted to get more involved with homebrew (see various cask version updates, adding Webots cask, this PR, etc. ) but decisions about bash seem to be out of scope even if this PR did initiate the discussion.